### PR TITLE
Repeat field is also compound types, ignore it.

### PR DESCRIPTION
**Pull request by baibaichen**
### Changelog category (leave one):
- Bug Fix (user-visible misbehavior in an official stable release)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Minor patch for https://github.com/ClickHouse/ClickHouse/pull/68131

For parquet written by spark,  `repeat` type is used for `Array`,  for example:

```thrift
message 68131_scheme {
    repeated int32 f
}
```

We also need ignore it.  otherwise excepion is thrown at `convertFieldToType`

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
- [ ]  Exclude: AST fuzzers
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by baibaichen** [comment]
@al13n321 please review it.

**Comment by alexey-milovidov** [comment]
The changelog entry has to be rewritten for better clarity.

File: src/Processors/Formats/Impl/ParquetBlockInputFormat.cpp [hunk 1]
```diff
@@ -286,6 +286,9 @@ static std::vector<Range> getHyperrectangleForRowGroup(const parquet::FileMetaDa
         if (!s)
             continue;
 
+        if(s->descr()->schema_node()->is_repeated())
```
Comment (alexey-milovidov) [inline src/Processors/Formats/Impl/ParquetBlockInputFormat.cpp:289]: Missing whitespace.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
Needs a test.

File: src/Processors/Formats/Impl/ParquetBlockInputFormat.cpp (hunk 1 above)
Comment (baibaichen) [reply src/Processors/Formats/Impl/ParquetBlockInputFormat.cpp]: fix

**Comment by baibaichen** [comment]
> Needs a test.

@alexey-milovidov test is added.

**Comment by baibaichen** [comment]
@alexey-milovidov @al13n321 would you please review this PR? 
Thanks

**Review by al13n321 — APPROVED** [review]
Thanks!

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit 858f8b502002661584e6153d39a23edc87b49dda with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/68494/858f8b502002661584e6153d39a23edc87b49dda/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/67bbe6fe98c41958067bf68370232a0eb24af1f7/docs_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

**Comment by al13n321** [comment]
@alexey-milovidov , please approve.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**